dts/arm/st/g4/stm32g431.dtsi

```diff
@@ -1,7 +1,27 @@
 /*
- * Copyright (c) 2019 Richard Osterloh <richard.osterloh@gmail.com>
+ * Copyright (c) 2019 STMicroelectronics.
```

Comment:
Not sure this change is valid.

Reply:
Also a copied file, the "original" empty file is pretty much gone.  I never really know what to do in causes like that.